spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/convert/MappingSpannerConverter.java

```diff
@@ -45,59 +46,67 @@
  * @author Balint Pato
  * @author Chengyuan Zhao
  */
-public class MappingSpannerConverter extends AbstractSpannerCustomConverter
-		implements SpannerConverter {
-
-	private static final Collection<Converter> DEFAULT_SPANNER_CONVERTERS = ImmutableSet
-			.<Converter>builder()
-			.addAll(SpannerConverters.DEFAULT_SPANNER_WRITE_CONVERTERS)
-			.addAll(SpannerConverters.DEFAULT_SPANNER_READ_CONVERTERS).build();
-
+public class MappingSpannerConverter implements SpannerConverter {
```

Comment:
I think we should rename `SpannerConverter` and `MappingSpannerConverter` at this point. We're pretty heavily invested in not having "converters" do the reading or writing, so maybe the names could be `SpannerCorrespondent` and `MappingSpannerCorrespondent`